nx-X11/programs/Xserver/randr/rrdispatch.c

```diff
@@ -21,48 +21,62 @@
  */
 
 #include "randrstr.h"
-
-#define SERVER_RANDR_MAJOR	1
-#define SERVER_RANDR_MINOR	2
+#ifndef NXAGENT_SERVER
```

Comment:
This is true. I had discussed that with @Ionic earlier while he backported the Composite extension code (0.2 -> 0.4) to nx-libs.

His reasoning was that NoMachine used NXAGENT_SERVER for indicating missing Xserver features.

I agree that we should have some XORG_TOO_OLD macro or such, as well. I won't do this for this pull request but may provide a patch for that later. See #144.